prow/cmd/deck/static/api/pretty-print.ts

```diff
@@ -0,0 +1,39 @@
+export type Language =
```

Comment:
Since this is not actually part of our internal API, but instead type definitions for something else being pulled in, I think it should live somewhere else - perhaps just next to `plugin-help.ts`.